src/Rubric/RubAbstractTextArea.class.st

```diff
@@ -991,6 +992,10 @@ RubAbstractTextArea >> handleEdit: editBlock [
 	"Ensure that changed areas get suitably redrawn"
 
 	editBlock value.
+	"Fit to width on edit"
+	(self grow isNotNil & self grow) ifTrue: [
```

Comment:
I will integrate it but I would have preferred to have grow initialized to false and to avoid the isNil check.